tests/self/README.md

```diff
@@ -0,0 +1,3 @@
+# Self tests
+
+Each of this files should container test probes for the standard library.
```

Comment:
```suggestion
Each of these files should contain only test probes for the standard library.
```

Reply:
I'll make this change ahead of submitting, if the rest looks good plz approve and I will clean up and submit together.